src/app/zap-templates/templates/app/MatterIDL.zapt

```diff
@@ -79,9 +79,11 @@
 
 {{/all_user_clusters}}
 
-{{#user_endpoints}}
+{{#chip_endpoints}}
 endpoint {{endpointId}} {
-    {{!TODO: report device types and cluster instantionation. }}
+  {{#chip_endpoint_clusters}}
+  instantiate cluster {{asUpperCamelCase name}};
```

Comment:
```suggestion
  instantiate {{side}} cluster {{asUpperCamelCase name}};
```
in that it's worth telling apart servers and clients, I suspect.

Reply:
Do they have any meaning here?

I thought our endpoint info is always server-side: it means what my device exposes to the  world.
When I create a client, I will not be able to tell on which  endpoint a lock/light/other actually resides, so saying "endpoint 5 { instantiate client cluster Lock }" seems odd to me. If I need to control a light, I should query the remote  side for what endpoints it exposes.

Reply:
I will follow up offline and do this in a followup if it is useful for understanding clusters.